ext/ftp/tests/ftp_delete_basic.phpt

```diff
@@ -0,0 +1,32 @@
+--TEST--
+Testing ftp_delete basic
+--CREDITS--
+Ward Cappelle <wardcappelle@gmail.com>
+User Group: PHP-WVL & PHPGent #PHPTestFest
+--SKIPIF--
+<?php
+require 'skipif.inc';
+?>
+--FILE--
+<?php
+require 'server.inc';
+
+$ftp = ftp_connect('127.0.0.1', $port);
+if (!$ftp) die("Couldn't connect to the server");
+ftp_login($ftp, 'user', 'pass');
+
+echo "Test case #1: removal of existing file from FTP, should return true:", PHP_EOL;
+var_dump(ftp_delete($ftp, 'file1'));
+
+echo "Test case #2: removal of non-existent file from FTP, should return false:", PHP_EOL;
+var_dump(ftp_delete($ftp, 'false-file.boo'));
+
+ftp_close($ftp);
+?>
+--EXPECTF--
+Test case #1: removal of existing file from FTP, should return true:
+bool(true)
+Test case #2: removal of non-existent file from FTP, should return false:
+
+Warning: ftp_delete(): No such file or directory in %s on line %d
```

Comment:
The `ftp_delete` function is been tested via https://github.com/php/php-src/blob/master/ext/ftp/tests/ftp_delete.phpt, but it does not cover this. Maybe you could edit that file? :)

Reply:
Thank you on this follow-up! I just set up a new PR to re-apply the same test to the newer ftp_delete.phpt-file:
https://github.com/php/php-src/pull/3814